src/entities/Proposal/routes.ts

```diff
@@ -339,6 +340,31 @@ export async function createProposalGrant(req: WithAuth) {
   })
 }
 
+const newProposalLinkedWearablesValidator = schema.compile(newProposalLinkedWearablesScheme)
+export async function createProposalLinkedWearables(req: WithAuth) {
+  const user = req.auth!
+  const configuration = validate<NewProposalLinkedWearables>(newProposalLinkedWearablesValidator, req.body || {})
+
+  req.body.links = req.body.links.map((link: string) => {
+    if(link.startsWith('http')) {
+      return link
+    }
+    
+    return 'https://' + link
```

Comment:
I think you can add this validation in the input field
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url